components/builder-web/app/actions/packages.ts

```diff
@@ -101,6 +111,20 @@ export function filterPackagesBy(
     };
 }
 
+export function populateDashboardRecent(data) {
+    let grouped = groupBy(data.results, "name");
+    let mapped = [];
+
+    for (let k in grouped) {
```

Comment:
It wouldn't be terrible if it were omitted in this case, but the `let` just keeps the `k` scoped to the loop; otherwise, `k` would spill out of the loop with whatever its value was on the last iteration.